src/components/grid-aware/ThreeParagraphBlock/ThreeParagraphBlock.js

```diff
@@ -61,7 +61,13 @@ const ParagraphBlock = ({ title, description, button }) => {
 ParagraphBlock.propTypes = {
   title: PropTypes.string.isRequired,
   description: PropTypes.string.isRequired,
-  button: PropTypes.shape(Button.propTypes),
+  button: PropTypes.shape({
+    text: PropTypes.string.isRequired,
+    noHover: PropTypes.bool,
+    externalLink: PropTypes.string.isRequired,
+    internalLink: PropTypes.string.isRequired,
+    onClick: PropTypes.func.isRequired,
+  }),
```

Comment:
You'll actually have to inline the original definition that had the `PropTypes.oneOf()` expression in there, or else everything will fail this because you've marked all the fields as required.

If you want a quicker way to run the tests without having to push code up to GitHub, you should be able to just run them with `npm run lint` locally.

Reply:
`const ParagraphPropType = PropTypes.shape({
  title: PropTypes.string.isRequired,
  description: PropTypes.string.isRequired,
  button: PropTypes.oneOfType([
    PropTypes.exact({
      text: PropTypes.string,
      externalLink: PropTypes.string,
    }),
    PropTypes.exact({
      text: PropTypes.string,
      internalLink: PropTypes.string,
    }),
    PropTypes.exact({
      text: PropTypes.string,
      onClick: PropTypes.func,
    }),
  ]),
});
#`

Reply:
const ParagraphPropType = PropTypes.shape({
  title: PropTypes.string.isRequired,
  description: PropTypes.string.isRequired,
  button: PropTypes.oneOfType([
    PropTypes.exact({
      text: PropTypes.string,
      externalLink: PropTypes.string,
    }),
    PropTypes.exact({
      text: PropTypes.string,
      internalLink: PropTypes.string,
    }),
    PropTypes.exact({
      text: PropTypes.string,
      onClick: PropTypes.func,
    }),
  ]),
});

Reply:
> const ParagraphPropType = PropTypes.shape({
  title: PropTypes.string.isRequired,
  description: PropTypes.string.isRequired,
  button: PropTypes.oneOfType([
    PropTypes.exact({
      text: PropTypes.string,
      externalLink: PropTypes.string,
    }),
    PropTypes.exact({
      text: PropTypes.string,
      internalLink: PropTypes.string,
    }),
    PropTypes.exact({
      text: PropTypes.string,
      onClick: PropTypes.func,
    }),
  ]),
});

Reply:
> const ParagraphPropType = PropTypes.shape({
  title: PropTypes.string.isRequired,
  description: PropTypes.string.isRequired,
  button: PropTypes.oneOfType([
    PropTypes.exact({
      text: PropTypes.string,
      externalLink: PropTypes.string,
    }),
    PropTypes.exact({
      text: PropTypes.string,
      internalLink: PropTypes.string,
    }),
    PropTypes.exact({
      text: PropTypes.string,
      onClick: PropTypes.func,
    }),
  ]),
});

Reply:
So above is the original definition with PropTypes.oneOf() expression, do I add the whole expression or just add the properties text, internalLink, externalLink, and onClick?

> ParagraphBlock.propTypes = {
  title: PropTypes.string.isRequired,
  description: PropTypes.string.isRequired,
  button: PropTypes.shape({
    text: PropTypes.string,
    externalLink: PropTypes.string,
    internalLink: PropTypes.string,
    onClick: PropTypes.func,
  }),
};

Reply:
@richardxia thank you in advance

Reply:
It's a little bit hard to read, but I think you'll want to do what you had in your first version.

BTW, when you want to paste a snippet of code in a comment, you should use triple backticks (```) so that the indentation is preserved and so that the text gets formatted in a monospaced font. See https://docs.github.com/en/github/writing-on-github/creating-and-highlighting-code-blocks#fenced-code-blocks for the documentation about this.